src/domains/messages-app/01_network.tf

```
@@ -76,6 +76,12 @@ data "azurerm_subnet" "azdoa_snet" {
   resource_group_name  = local.vnet_common_resource_group_name
 }
 
+data "azurerm_subnet" "github_snet" {
```

Do you mind adding the backend l3 snet to the whitelisted subnets?

```suggestion
data "azurerm_subnet" "app_backendl3_snet" {
  name                 = "appbackendl3"
  virtual_network_name = local.vnet_common_name
  resource_group_name  = local.vnet_common_resource_group_name
}

data "azurerm_subnet" "github_snet" {
```